porch/func/Dockerfile-wrapperserver

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.17.8-alpine3.15 as builder
+
+COPY . ./func
+RUN cd ./func ; go mod download
+RUN cd ./func ; go build -o /usr/local/bin/wrapper-server ./wrapper
+
+FROM alpine:3.15
+
+COPY --from=builder /usr/local/bin/wrapper-server /usr/local/bin/wrapper-server
+ENTRYPOINT cp /usr/local/bin/wrapper-server /wrapper-server/wrapper-server
```

Comment:
we may want to take the destination as a parameter to not have the contract hardcoded between the dockerfile and the service spinning up the pod